.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,1219 @@
+
```

Comment:
This file doesn't look right at all. I guess there should be Markdown and no HTML here.